### PR TITLE
add raster type conversion operator

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[Raster Type Conversion Operator](https://trello.com/c/GWZCtJy4/963-raster-type-conversion-operator)

File: package.json [hunk 1]
```diff
@@ -110,7 +110,6 @@
         "karma-jasmine": "~5.1.0",
         "karma-jasmine-html-reporter": "~2.0.0",
         "ng-packagr": "^14.1.0",
-        "@nrwl/devkit": "// remove this when <https://github.com/nrwl/nx/issues/11653> is resolved",
```
Comment (ChristianBeilschmidt) [inline package.json:113]: No, you would need to remove `@nrwl/devkit` completely and not just the comment :smile:

File: projects/wave-core-new/src/lib/operators/dialogs/raster-type-conversion/raster-type-conversion.component.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import {RasterLayer} from '../../../layers/layer.model';
+import {ResultTypes} from '../../result-type.model';
+import {AfterViewInit, ChangeDetectionStrategy, Component, OnDestroy, OnInit} from '@angular/core';
+import {FormControl, UntypedFormBuilder, UntypedFormGroup, Validators} from '@angular/forms';
```
Comment (ChristianBeilschmidt) [inline projects/wave-core-new/src/lib/operators/dialogs/raster-type-conversion/raster-type-conversion.component.ts:4]: When you create a new operator, please use typed forms to not add more technical debts.

File: projects/wave-core-new/src/public-api.ts [hunk 3]
```diff
@@ -72,6 +72,7 @@ export * from './lib/operators/dialogs/raster-vector-join/raster-vector-join.com
 export * from './lib/operators/dialogs/scatterplot-operator/scatterplot-operator.component';
 export * from './lib/operators/dialogs/statistics-plot/statistics-plot.component';
 export * from './lib/operators/dialogs/temporal-raster-aggregation/temporal-raster-aggregation.component';
+export * from './lib/operators/dialogs/raster-type-conversion/raster-type-conversion.component';
```
Comment (ChristianBeilschmidt) [inline projects/wave-core-new/src/public-api.ts:75]: This list was sorted lexicographically so please sort it again.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]
This CI is failing.

File: package.json (hunk 1 above)
Comment (jdroenner) [reply package.json:113]: npm will remove the dup entry every time. i did not reverse that.

File: projects/wave-core-new/src/public-api.ts (hunk 3 above)
Comment (jdroenner) [reply projects/wave-core-new/src/public-api.ts:75]: why is there no auto sort if that is important?

File: projects/wave-core-new/src/lib/operators/dialogs/raster-type-conversion/raster-type-conversion.component.ts (hunk 2 above)
Comment (jdroenner) [reply projects/wave-core-new/src/lib/operators/dialogs/raster-type-conversion/raster-type-conversion.component.ts]: [04e7ba2](https://github.com/geo-engine/geoengine-ui/pull/424/commits/04e7ba211b8feffab53221b5fa3149d7db26b67f)

File: projects/wave-core-new/src/public-api.ts (hunk 3 above)
Comment (jdroenner) [reply projects/wave-core-new/src/public-api.ts]: [426cf2e](https://github.com/geo-engine/geoengine-ui/pull/424/commits/426cf2ee8f30ee0443486e2581c97fe1b701909c)

Comment (ChristianBeilschmidt) [reply projects/wave-core-new/src/public-api.ts]: I'm not sure how to do that.

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**